internal/database/repositories/org_repository.go

```diff
@@ -41,3 +41,9 @@ func (g *orgRepository) ReadBySlug(slug string) (models.Org, error) {
 	err := g.db.Where("slug = ?", slug).First(&t).Error
 	return t, err
 }
+
+func (g *orgRepository) GetSlug(orgID string) (models.Org, error) {
```

Comment:
this does not make too much sense. The function is called GetSlug and returns a whole model. I think you can just use the "Read" method